de.klayout.KLayout.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: de.klayout.KLayout
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: klayout
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --filesystem=host
+- --device=dri
+modules:
+- name: gdbm
+  config-opts:
+  - --enable-libgdbm-compat
+  sources:
+  - type: archive
+    url: ftp://ftp.gnu.org/gnu/gdbm/gdbm-1.18.1.tar.gz
+    sha256: 86e613527e5dba544e73208f42b78b7c022d4fa5a6d5498bf18c8d6f745b91dc
+- name: ruby
+  config-opts:
+  - --enable-shared
+  sources:
+  - type: archive
+    url: https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.gz
+    sha256: d418483bdd0000576c1370571121a6eb24582116db0b7bb2005e90e250eae418
+- name: klayout
+  buildsystem: simple
+  build-commands:
+  - bash ./build.sh -bin ${FLATPAK_DEST}/bin -j$FLATPAK_BUILDER_N_JOBS
+  post-install:
+  - install -Dm644 de.klayout.KLayout.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml
+  - install -Dm644 etc/klayout.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+  - install -Dm644 etc/logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+  - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+  sources:
+  - type: archive
+    url: https://www.klayout.org/downloads/source/klayout-0.26.5.tar.gz
+    sha256: 3f58be531b5570e9d01ac8077a84b90b91fe181e01dd4acc505c8a77dcdc8d88
+  - type: file
+    path: de.klayout.KLayout.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Okay, I switched out the appdata file for a metainfo file.